examples/create_geopandas_from_pandas.py

```diff
@@ -28,21 +27,15 @@
      'Longitude': [-58.66, -47.91, -70.66, -74.08, -66.86]})
 
 ###############################################################################
-# A ``GeoDataFrame`` needs a ``shapely`` object, so we create a new column
-# **Coordinates** as a tuple of **Longitude** and **Latitude** :
+# A ``GeoDataFrame`` needs a ``shapely`` object. We use geopandas
+# ``points_from_xy()`` to transform **Longitude** and **Latitude** into a list
+# of ``shapely.Point`` objects and set it as a ``geometry`` while creating the
+# ``GeoDataFrame``.
 
-df['Coordinates'] = list(zip(df.Longitude, df.Latitude))
+gdf = geopandas.GeoDataFrame(df,
+                             geometry=geopandas.points_from_xy(df.Longitude,
```

Comment:
style wise nit: I would put the `df` already on the next line, then the `points_from_xy` doesn't need another overflow:

```
gdf = geopandas.GeoDataFrame(
    df, geometry=geopandas.points_from_xy(df.Longitude, df.Latitude))
```